index.html

```diff
@@ -6,11 +6,56 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Pacifico&display=swap" rel="stylesheet">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <img id="header-image" src="https://i.postimg.cc/Zq6Jqk9D/2a6c1858-0fe8-48a7-82fe-1976833b8691-200x200.png" alt="Let Them Eat Cake logo">
+
+        <nav class="navigation">
+            <ul>
+                <li><a href="">Home</a></li>
+                <li><a href="">Cakes</a></li>
+                <li><a href="">Ordering</a></li>
+                <li><a href="">Lessons</a></li>
+                <li><a href="">About</a></li>
+            </ul>
+
+        </nav>
+    </header>
+
+    <main>
+        
+        <section class="about">
+            <h1>Let Them Eat Cake</h1>
+
+            <h3>Feeding the cake deprived since 2020</h3>
+        </section>
+
+        <section class="image-carousel">
```

Comment:
Keep going @melanieet 😉 
You don't need to make a real carousel but you can fake it with a large image and some smaller 'thumbnails' underneath